server/src/main/java/com/scalar/db/server/ServerModule.java

```diff
@@ -1,4 +1,4 @@
-package com.scalar.db.server.service;
+package com.scalar.db.server;
```

Comment:
I also moved `ServerModule` from the `com.scalar.db.server.service` package to the `com.scalar.db.server` package.